src/models/regression.py

```diff
@@ -98,3 +117,26 @@ def predict(self) -> Tuple[Dict[str, Dict[str, np.ndarray]], Dict[str, np.ndarra
                 test_arrays_dict[key] = {'y': val.y, 'latlons': val.latlons}
 
         return test_arrays_dict, preds_dict
+
+    def _calculate_big_mean(self) -> np.ndarray:
```

Comment:
what does the `_big_mean` do / refer to?

Reply:
as in why do we calculate it?

Reply:
For the linear explainer, the mean of the dataset is necessary - it's passed to it when we initialize the explainer.